dts/bindings/gpio/nxp,pca6416.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Base binding for PCA6416 I2C-based GPIO expander
+
+compatible: "nxp,pca6416"
+
+include: [gpio-controller.yaml, i2c-device.yaml]
+
+properties:
+  interrupt-gpios:
```

Comment:
Why do you have 2 of these `interrupt-gpios` and `int-gpios`

Reply:
removed int-gpios, thanks.